docker/docker-compose.yml

```diff
@@ -256,6 +256,7 @@ services:
       MF_JAEGER_URL: ${MF_JAEGER_URL}
     ports:
       - ${MF_COAP_ADAPTER_PORT}:${MF_COAP_ADAPTER_PORT}/udp
+      - ${MF_COAP_ADAPTER_PORT}:${MF_COAP_ADAPTER_PORT}
```

Comment:
Why remove udp?

Reply:
I didn't remove UDP, I just exposed TCP, so that we can access version and metrics endpoints.